tests/test_start.py

```diff
@@ -264,6 +264,7 @@ async def test_over_switch_deactivate_preset(
             CONF_HEATER_2: None,
             CONF_HEATER_3: None,
             CONF_HEATER_4: None,
+            CONF_HEATER_KEEP_ALIVE: 0,
```

Comment:
Please add a unit test with KEEP_ALIVE not 0 to see if method are called.

You can inspire from the existing tests `test_bug_66` which mocks method, check they are called and waits for the window to be open/close.

Let me know if it is too complex I'm pretty sure I can help.